### PR TITLE
Feat/RSS-ECOMM-1_07 Set up commerce tools API client

### PR DESCRIPTION
**Pull request by AndreiMakhnach2209**
### What type of PR is this?

- [x] 🌟 Feature
- [ ] 🛠 Bug Fix
- [ ] ⚙️ Refactor
- [ ] 📖 Documentation

### Description

- **Brief Overview:**
Set up commerce tools API client with Axion.

### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: src/api/API.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            username: process.env.CTP_CLIENT_ID ?? "",
+            password: process.env.CTP_CLIENT_SECRET ?? "",
+          },
+        }
+      );
+      if (response.status === 200) {
+        const accessToken = response.data.access_token;
+        const tokenType = response.data.token_type;
+        localStorage.setItem("ACCES_TOKEN", accessToken);
+        localStorage.setItem("TOKEN_TYPE", tokenType);
```
Comment (Oleg-Melnikow) [inline src/api/API.tsx]: Can we use Destructuring assignment like this: 

`
const { token_type,  access_token } = response.data;
`

and save localStorage object with tokens, for example
`
localStorage.setItem("APP_TOKENS", JSON.stringify( { token_type,  access_token } ) );
`

**Review by Oleg-Melnikow — COMMENTED** [review]
Good job! Please check the comment, maybe it will improve your code.

File: src/api/API.tsx (hunk 1 above)
Comment (AndreiMakhnach2209) [reply src/api/API.tsx]: According to the rules of our project, there is no possibility of a destructured assignment in this case.
`Variable name must match one of the following formats: camelCase, PascalCase, UPPER_CASE (@typescript-eslint/naming-convention)`

Comment (Oleg-Melnikow) [reply src/api/API.tsx]: maybe like this:
`const { token_type: accessToken,  access_token: tokenType } = response.data;`
and save this object `{ accessToken,  tokenType  }` in localStorage

**Review by JeckJonnyQ — APPROVED** [review]

**Pull request merged**